examples/echo_server/migrations/6_transfer_link.js

```diff
@@ -2,11 +2,11 @@ let LinkToken = artifacts.require("../node_modules/smartcontractkit/chainlink/so
 let RunLog = artifacts.require("./RunLog.sol");
 let devnetAddress = "0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f";
 
-module.exports = async function(deployer) {
-  await LinkToken.deployed().then(async function(linkInstance) {
+module.exports = function(deployer) {
```

Comment:
Apparently truffle migrations don't support `async` at the top export level, but they do inside `then(async....`:
https://github.com/trufflesuite/truffle/issues/501

Reply:
Maybe still return result of LinkToken.deployed? Not sure about Truffle's internals, but in tests it waits for returned promises to finish.

Reply:
I tried that to no avail and felt this was the most confident solution.